blog-homepage/src/your-articles.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import yourArticlesData from './your-articles.json';
+
+function LinkedImage(props) {
+  return(
+    <a className='your-article link' href={props.source.link}>
+      <img className='your-article image' src={props.source.image} alt={props.source.image}/>
+    </a>
+  )
+}
+
+function TitleDescriptionBlock(props){
+  return(
+    <div>
+     <h2 className='your-article title'>{props.source.title}</h2>
+     <div className='your-article description'>{props.source.description}</div>
+    </div>
+  )
+}
+
+function AuthorBio(props) {
```

Comment:
Is this the same `AuthBio` as in `missed-articles.js`? Consider refactoring and importing to each.